src/IndicatorsDash.jsx

```diff
@@ -95,7 +95,7 @@ class IndicatorsDash extends React.Component {
             size={item.size}
             name={item.name}
             values={item.values}
-            areaName={areaName}
+            activeArea={activeArea}
```

Comment:
An error is occurring when you reload the dash component since you are ignoring areaName, and activeArea is not always ready.